src/vm/amd64/jithelpers_fast.S

```diff
@@ -33,8 +33,13 @@ LEAF_ENTRY JIT_CheckedWriteBarrier, _TEXT
         // See if this is in GCHeap
         PREPARE_EXTERNAL_VAR g_lowest_address, rax
         cmp     rdi, [rax]
+#ifdef FEATURE_WRITEBARRIER_COPY
+        // jb      NotInHeap
+        .byte 0x72, 0x12
+#else
```

Comment:
I'm curious, why can this not use a mnemonic?

Reply:
The reason is that the clang assembler generates the long version of the relative jump instruction instead of one with byte offset for some reason. We've tried various things to make it generate this short version, but never found a way. So we had to resort to this ugly way.